Recast/SeriesViewController.swift

```diff
@@ -18,6 +18,9 @@ class SeriesViewController: UIViewController {
     // MARK: - Constants
     let episodeCellReuseIdentifer = "episodeCell"
 
+    var partialPodcast: PartialPodcast!
```

Comment:
what are these changes for

Reply:
I'm not sure I was going to ask you if I can delete this file. These changes were to fix a build error I think.